browser/ui/views/commands/accelerator_service_factory.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_COMMANDS_ACCELERATOR_SERVICE_FACTORY_H_
+#define BRAVE_BROWSER_UI_VIEWS_COMMANDS_ACCELERATOR_SERVICE_FACTORY_H_
+
+#include "base/memory/singleton.h"
+#include "brave/browser/ui/views/commands/accelerator_service.h"
+#include "components/keyed_service/content/browser_context_keyed_service_factory.h"
+#include "components/keyed_service/core/keyed_service.h"
+#include "content/public/browser/browser_context.h"
+
+namespace commands {
+
+class AcceleratorServiceFactory : public BrowserContextKeyedServiceFactory {
```

Comment:
please use `ProfileKeyedServiceFactory` with `ProfileSelection::kRedirectedToOriginal`.

Reply:
Awesome, I didn't know about this - that's really handy.